### PR TITLE
code normalization and fix PSR-2

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: live-top10-memory-usage/src/table_top10memory.ihtml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 	</td>
         <td class="ListColLeft">
 	 <a href="/centreon/main.php?p=20201&amp;o=svcd&amp;host_name={$elem.host_name}&service_description={$elem.service_description}" target="_blank">{$elem.host_name}</a>
```
Comment (sc979) [inline live-top10-memory-usage/src/table_top10memory.ihtml]: wrong indentation

File: live-top10-memory-usage/src/top10_memory.css [hunk 2]
```diff
@@ -8,26 +8,26 @@
   border-radius: 4px;
   }
   .bar .progress-ok {
-  background-color: #caef74;                                                                                                                                                            
+  background-color: #caef74;
```
Comment (sc979) [inline live-top10-memory-usage/src/top10_memory.css]: wrong indentation

**Comment by sc979** [comment]
Hi @lgcosta ,
Thanks for your PR, but could you check the coding style rules ;)
`https://github.com/centreon/centreon/tree/19.04.x/doc/coding-style/css`
sc979

**Comment by lgcosta** [comment]
> Hi @lgcosta ,
> Thanks for your PR, but could you check the coding style rules ;)
> `https://github.com/centreon/centreon/tree/19.04.x/doc/coding-style/css`
> sc979

Yes, until then I was not seeing other "non-PHP" files. I will see that too. Thanks again !

**Comment by lgcosta** [comment]
thanks @sc979, is done.

Now, i will review the other repositories to fix the html and others files.

File: live-top10-memory-usage/configs.xml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+    <email>contact@centreon.com</email>
+    <website>http://www.centreon.com</website>
+    <description>This widget lists hosts ordered by memory usage (percentage). It is possible to select the number of lines in the table or to select a subset of resources.</description>
+    <version>19.10.0</version>
+    <keywords>centreon, widget, memory</keywords>
+    <screenshot></screenshot>
+    <thumbnail>./widgets/live-top10-memory-usage/resources/logo1.png</thumbnail>
+    <url>./widgets/live-top10-memory-usage/index.php</url>
+    <autoRefresh>10</autoRefresh>
 	<preferences>
```
Comment (sc979) [inline live-top10-memory-usage/configs.xml]: could you align / correct the indentation of this line too please

Comment (lgcosta) [reply live-top10-memory-usage/configs.xml]: "maledito" tab key... :triumph:
thanks !

File: live-top10-memory-usage/src/table_top10memory.ihtml [hunk 4]
```diff
@@ -1,67 +1,79 @@
...
+        </table>
+        </div>
+    {else if $preferences.service_description == '' && $preferences.metric_name == '' && $preferences.nb_lin == ''}
+        <center>
+            <div class="update" style="text-align:center;width:360px;">
+                Please select a metric_name, nb_lin, service_description
+            </div>
+        </center>
+    {/if}
 	<script>
```
Comment (jdelpierre) [inline live-top10-memory-usage/src/table_top10memory.ihtml]: wrong indentation ;)

Comment (jdelpierre) [inline live-top10-memory-usage/src/table_top10memory.ihtml]: wrong indentation ;)

Comment (lgcosta) [reply live-top10-memory-usage/src/table_top10memory.ihtml]: > wrong indentation ;)

"maledito" tab indentation here and a better flux of block indentation. Thanks

**Pull request merged**